src/test/java/com/cisco/mongodb/aggregate/support/test/repository/ArtworkRepository.java

```diff
@@ -154,4 +154,20 @@
     return getFacetResultsWithConditional(showTags, false, false);
   }
 
+  @Aggregate2(inputType = Artwork.class, genericType = true, outputBeanType = HashMap.class)
+  @Facet2(pipelines = {
+      @FacetPipeline(name = "limitPipeline",
+                     stages = {
+                         @FacetPipelineStage(stageType = Limit2.class, query = "2",
+                                             condition = {
```

Comment:
Add an attribute called conditionMatchType = ANY|ALL (default to ANY) where ANY and ALL are defined as Enums inside the Conditional annotation